prime-router/src/main/kotlin/fhirengine/utils/HL7MessageHelpers.kt

```diff
@@ -103,4 +103,8 @@ object HL7MessageHelpers : Logging {
 
         return builder.toString()
     }
+
+    fun messageCount(rawHl7: String): Int {
```

Comment:
This function should be in a companion object; that way, a new instance of `HL7MessageHelpers` doesn't need to be instantiated every time this function is called.

Reply:
HL7MessageHelpers is an object. As far as I understand it, classes can have companion objects, but objects cannot.